library/std/src/f64.rs

```diff
@@ -377,6 +411,10 @@ impl f64 {
     ///
     /// Returns NaN if `self` is a negative number other than `-0.0`.
     ///
+    /// # Platform-specific precision
+    ///
+    /// The precision of this function varies by platform and Rust version.
```

Comment:
sqrt is defined by the IEEE standard and should always produce the same result on all platforms.

Reply:
Interesting, I didn't know that.

Reply:
Fixed.